project/settings/development.py

```
@@ -21,27 +22,50 @@
     }
 }
 
+STORAGES = {
+    "default": {
+        "BACKEND": "storages.backends.s3boto3.S3Boto3Storage",
+        "OPTIONS": {
+            "bucket_name": "mindjunkies",
+            "access_key": config("AWS_ACCESS_KEY_ID"),
```

Ruff (0.8.2)

`config` may be undefined, or defined from star imports

(F405)